lib/components/Dashboard.js

```diff
@@ -37,29 +37,58 @@ export default class Dashboard extends PureComponent {
       customChildrenBefore,
       primaryWidget,
       bottomWidgets = [],
-      customChildrenAfter
+      customChildrenAfter,
     } = this.props;
     if (this.state.hasError)
       return <Text type="p">There was a widget error</Text>;
+    const hasWidgets =
+      primaryWidget ||
+      bottomWidgets.length ||
+      customChildrenBefore ||
+      customChildrenAfter;
     return (
       <div className="dashboard-container container">
         <Header type="h2">bPanel Dashboard</Header>
-        <Text type="p">
-          A simple Dashboard view that can be decorated by widgets. Widgets
-          allow you to extend your dashboard according to your needs.
-        </Text>
-        <Text type="p">
-          Everything in this view is its own widget and can be edited, removed,
-          and rearranged as desired.
-        </Text>
+        {hasWidgets ? (
+          ''
+        ) : (
+          <div>
+            <Text type="p">
+              This is a simple Dashboard view that can be decorated by widgets.
+              Widgets allow you to extend your dashboard according to your
+              needs.
+            </Text>
+            <Text type="p">
+              In order to add widgets to this view, you can install any
+              compatible widget plugin, or create your own. This allows each
+              user to compose their own customized dashboard view! Once you've
+              installed your first widget, this intro text will disappear.
+            </Text>
+            <Text type="p">Some widgets we recommend to get started:</Text>
+            <ul>
+              <li>@bpanel/recent-blocks</li>
```

Comment:
Would it be useful to keep these `li`s in an array and map over them so that it is easier to update/change these in the future?

Reply:
Feels like overkill to me for something that will rarely change if at all (this is just a prompt to get people to search anyway). Might actually turn into more code actually written.